async/cohttp_async.mli

```diff
@@ -20,18 +20,18 @@ open Async.Std
 
 (** Read in a full body and convert to a [string] *)
 
-module IO : Cohttp.S.IO with type 'a t = 'a Deferred.t
+module IO : (module type of Cohttp_async_io)
 
 module Request : sig
   type t = Cohttp.Request.t
   include Cohttp.S.Request with type t := Cohttp.Request.t
-  include Cohttp.S.Http_io with type t := Cohttp.Request.t and module IO=IO
```

Comment:
Why do you need this and the change below? Shouldn't changing `module IO` be enough?